README.md

```diff
@@ -1 +1,6 @@
 # vector-tiles-rasterizer
+
+a proxy server written in `fastify` for rasterizing vector tiles in real time.
+the vector tiles can be served from any vector tiles backend e.g. `tegola` or `tileserver-gl`.
+
+see [example](example/README.md) for additional information
```

Comment:
on the contrary, this is like a quick start for someone with not much background or programming knowledge. once the main readme will be more robust like in other services it will make more sense. @syncush wdyt?

Reply:
I agree with @melancholiai, README.md for a quick start can be under the example